apps/api-gateway/src/oid4vc-issuance/dtos/oid4vc-issuer-template.dto.ts

```
@@ -229,8 +229,11 @@ export class CreateCredentialTemplateDto {
   @IsEnum(CredentialFormat)
   format: CredentialFormat;
 
-  @ValidateIf((o: CreateCredentialTemplateDto) => CredentialFormat.SdJwtVc === o.format)
-  @IsEmpty({ message: 'doctype must not be provided when format is "dc+sd-jwt"' })
+  @ValidateIf(
+    (o: CreateCredentialTemplateDto) =>
+      CredentialFormat.SdJwtVc === o.format || CredentialFormat.JwtVcJsonLd === o.format
```

ESLint

[error] Expected no linebreak before this expression.

(implicit-arrow-linebreak)